src/main/java/org/tikv/common/ConfigUtils.java

```diff
@@ -67,6 +67,8 @@
   public static final String TIKV_RAWKV_SCAN_TIMEOUT_IN_MS = "tikv.rawkv.scan_timeout_in_ms";
   public static final String TIKV_RAWKV_CLEAN_TIMEOUT_IN_MS = "tikv.rawkv.clean_timeout_in_ms";
   public static final String TIKV_BO_REGION_MISS_BASE_IN_MS = "tikv.bo_region_miss_base_in_ms";
+  public static final String TIKV_SLOW_LOG_THRESHOLD = "tikv.slow_log_threshold";
+  public static final String TIKV_SLOW_LOG_PRETTY = "tikv.slow_log_pretty";
```

Comment:
can we remove this at present? we can add it back in the future if we need to print slow log in other formats.

Reply:
sure